policy/testdata/pb/policy.yaml

```diff
@@ -13,8 +13,20 @@
 # limitations under the License.
 
 name: "pb"
+
+imports:
+   - name: google.expr.proto3.test.TestAllTypes.NestedEnum
+   - name: >
+       "google.expr.proto3.test.ImportedGlobalEnum"
```

Comment:
I'm actually kind of surprised the import for this works. My impression was that we would have to "prune" the additional indentation and the quotes that comes along with the import due to the multiline syntax. Is there something in the abbreviation mechanism that makes this possible?

Reply:
Thinking about this further -- we should remove the container setting here to properly test this, so that it's not accidentally resolving these imports through the explicitly set container: https://github.com/google/cel-go/blob/master/policy/testdata/pb/config.yaml#L16.

Reply:
I adjusted the container to remove `proto3` which will cause namespace resolution to fail and forces me to add another alias. The reason I've kept the container at all is to simplify test case generation.